File: mario.c
#include <stdio.h>
#include <cs50.h>

int main (void) 
{
    // prompt user to enter the correct value
    int height;
    do 
    {
        height = get_int ("Height:\n ");
    }
    while (height < 1 || height > 8); 
    
    // loop the amount of rows
    for (int row = 0; row < height; row++)
    {
        // print the amount of spaces equal to height - (row + 1)
        for (int spaces = height - row; spaces > 1; spaces--)  
        {
            printf (" ");
        }
        // print the amount of hashes equal to row + 1
        for (int hash = 0; hash < row + 1; hash++)
        {
            printf ("#");
        }
        // print 2 spaces
        printf ("  ");
        // print amount of hashes equal to row + 1
         for (int hash = 0; hash < row + 1; hash++)
        {
            printf ("#");
        }
        //new line
        printf ("\n");
            
    }

}

File: mario.py
from cs50 import get_int

while True:
    number = get_int("Enter height\n")
    if number > 0 and number < 9:
        break
for i in range(1, number + 1, 1):
    print(" " * (number - i), end = "")
    print("#" * i, end = "  ")
    print("#" * i)



